src/main/java/com/redhat/devtools/intellij/common/CommonConstants.java

```diff
@@ -10,6 +10,9 @@
  ******************************************************************************/
 package com.redhat.devtools.intellij.common;
 
+import com.intellij.openapi.util.Key;
+
 public class CommonConstants {
     public static final String HOME_FOLDER = System.getProperty("user.home");
+    public static final Key<String> KIND_PLURAL = Key.create("tekton.plural");
```

Comment:
I don't think that should be in common package as it seems to be Tekton specific